tests/bin/recover_batch_test.py

```diff
@@ -35,3 +44,28 @@ def test_notify(mock_reactor, mock_read_last_yaml_entries, line, exit_code, erro
     assert actual_exit_code == exit_code
     assert actual_error_msg == error_msg
     assert mock_reactor.stop.call_count == 1
+
+
+def test_read_last_yaml_roundtrip(mock_file):
+    """Check that read_last_yaml_entries returns the same thing that the action
+    runner wrote."""
+    status = StatusFile(mock_file)
+    expected_content = {'return_code': 1, 'pid': 10345, 'command': 'foo'}
```

Comment:
nit: can you provide an example where there are multiple lines, and have the test expect only the last?

Reply:
This test actually does write multiple lines, updated the test to check both lines so it's more clear.